src/modules/filesystem.md

```diff
@@ -41,10 +34,12 @@ pub fn harvest(garden: &mut Garden) { todo!() }
 
 <details>
 
-* The change from `module/mod.rs` to `module.rs` doesn't preclude the use of submodules in Rust 2018.
-  (It was mandatory in Rust 2015.)
+* Before Rust 2018, modules needed to be located at `module/mod.rs` instead of `module.rs`, and this is still a working alternative for current versions.
```

Comment:
```suggestion
* Before Rust 2018, modules needed to be located at `module/mod.rs` instead of `module.rs`, and this is still a working alternative for editions after 2018.
```